crates/biome_js_analyze/src/lint/nursery/no_secrets.rs

```diff
@@ -10,14 +10,28 @@ use regex::Regex;
 
 use std::sync::LazyLock;
 
+use biome_deserialize_macros::Deserializable;
+use serde::{Deserialize, Serialize};
+
 // TODO: Try to get this to work in JavaScript comments as well
 declare_lint_rule! {
     /// Disallow usage of sensitive data such as API keys and tokens.
     ///
     /// This rule checks for high-entropy strings and matches common patterns
-    /// for secrets, such as AWS keys, Slack tokens, and private keys.
+    /// for secrets, including AWS keys, Slack tokens, and private keys.
```

Comment:
Should we list explicitly what secrets we try to catch? Now the only way to know would be looking into source code or blindly guessing.

Reply:
Good point!

Reply:
Added.